vars/kubectl.groovy

```diff
@@ -5,6 +5,9 @@ def createConfigMap(String name, String namespace, files) {
             case String:
                 fromFileArgs.add("--from-file=${files}")
                 break
+            case GString:
```

Comment:
String,GString as I understand will not work, but this will
`case GString:
  case String:
          fromFileArgs.add("--from-file=${files}")
          break`

Reply:
yes, this is also acceptable.